google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -845,6 +848,15 @@ public static SetStorageClassLifecycleAction newSetStorageClassAction(
         return new SetStorageClassLifecycleAction(storageClass);
       }
 
+      /**
+       * Create a new {@code AbortIncompleteMPUAction}. An incomplete multipart upload will be
+       * aborted when the multipart upload meets the specified condition. Age is the only condition
+       * supported for this action.
```

Comment:
Recommend adding a link

Reply:
A link to the docs? I don't know if we have one yet. @BrennaEpp do we have a docs link?